### DISCUSSION TITLE
Callpeaks on multiome scRNA - scATAC object

### DISCUSSION THREAD

**Discussion by yamihn** [stuart-lab/signac#1171] [Q&A] [answered]
Hello, I'm interested in finding DARs using FindMarkers with Signac. I ran FindMarkers(object, group.by="treatment_timepoint_cluster", ident.1= "2_wt_d8", ident.2 = "2_ko_d8", min.pct= 0.05, latent.vars= "peak_region_fragments", assay= "ATAC), and I visualize the DARs in coverage plots but there are any differences. The fold changes are around 0.6 to -0.6. I don't understand why I dont see anything. My question Is: can I find peaks with macs2 grouping by conditions and then create a new assay to run findmarkers again skipping the integration steps of this assay? Or I Need to integrated the assay peaks to the multiomic object before trying to apply findmarkers? Thanks

**Comment by timoast** [comment, marked as answer]
0.6 is quite low, so this isn't that surprising.

Can you elaborate on how you processed the data?

> My question Is: can I find peaks with macs2 grouping by conditions and then create a new assay to run findmarkers again skipping the integration steps of this assay?

Yes you can

> Or I Need to integrated the assay peaks to the multiomic object before trying to apply findmarkers?

You should not be using integrated data for differential testing
